ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/disconnected/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-vsphere-upi-disconnected-workers-rhcos-rhel8-provision-chain.yaml

```diff
@@ -0,0 +1,49 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-upi-disconnected-workers-rhcos-rhel8-provision
+  steps:
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-conf
+  - ref: ipi-conf-vsphere-check
+  - ref: ipi-conf-vsphere-mirror-registry
+  - ref: mirror-images-by-oc-adm
+  - ref: ipi-conf-mirror
+  - ref: upi-conf-vsphere
+  - ref: upi-conf-vsphere-ova
+  - ref: upi-install-vsphere
+  - ref: ipi-install-times-collection
+  - ref: ipi-conf-vsphere-bastionhost
+  - ref: workers-rhel-vsphere-provision
+  - ref: workers-rhel-vsphere-provision-dns
+  - ref: workers-rhel-pre-hook-haproxy
+  - ref: workers-rhel-pre-hook-sshtunnel
+  - chain: workers-rhel
+  env:
+  - name: RHEL_WORKER_COUNT
+    default: 2
+    documentation: The number of rhel worker to be provisioned.
+  - name: RHEL_IMAGE
+    default: "qe-rhel-85-release"
```

Comment:
how about using RHEL-8.6 directly in the new job

Reply:
or could we use the `PLATFORM_VERSION` to assemble the expected `RHEL_IMAGE` name, so that we can update this parameter more easier? I remember on other public platforms, they also use the `PLATFORM_VERSION` to set the RHEL image name.

Reply:
yeah, I know above way and the image name was assemble like this at the beginning, but "RHEL_IMAGE" was introduced, because this way is convenient for user to easily overwrite this ENV var to launch rhel instance by using their own image if needed. And azure also use the same way currently.

Reply:
Yeah, we introduced `RHEL_IMAGE` on purpose to allow user override the image (named with any string) easily.

Reply:
correct, both ENV vars need to be updated if new rhel version is coming. I updated to RHEL-8.6 on vSphere two workflows.